package.json

```diff
@@ -2,19 +2,28 @@
   "name": "atom-select-list",
   "version": "0.7.2",
   "description": "A general-purpose select list for use in Atom packages",
-  "main": "./src/select-list-view.js",
+  "main": "lib/select-list-view.js",
+  "files": [
+    "lib/**/*"
+  ],
   "scripts": {
-    "test": "atom --test test"
+    "typescript": "tsc -p src/tsconfig.json || echo 'tsc is done'",
+    "build": "npm run typescript -s",
+    "test": "atom --test test",
+    "prepare": "npm run build"
```

Comment:
Is `prepare` a special command or one that has some sort of conventional use (like build, test, etc)? What is its use here other than an alias to build?

Reply:
Prepare runs after local installation and before publishing.

https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts